index.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  name: 'loader.js',
+
+  init: function() {
+    this.treePaths['vendor'] = 'lib';
+  },
+
+  included: function() {
+    this.app.import('vendor/loader.js', {
+      prepend: true
```

Comment:
This ensures loader is prepended, in theory other things could also ask to be prepended today but they would already be added before the loader.

Basically, things that are prepended to vendor should not assume the loader works (this is how it is today also)...